<launch>

  <arg name="image_raw_topic" default="/real_camera"/>
  <arg name="twist_cmd_topic" default="/android_input"/>
  <arg name="base_folder" default="/set1"/>

    <node name="write_data" pkg="cnn" type="write_data.py" output="screen">
      <param name="image_raw_topic" value="$(arg image_raw_topic)"/>
      <param name="twist_cmd_topic" value="$(arg twist_cmd_topic)"/>
      <param name="base_folder" value="$(arg base_folder)"/>
    </node>

</launch>